addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -310,6 +320,26 @@ class ChatWindow extends Component {
         }
     }
 
+    /**
+     * @private
+     */
+    async _onHideHomeMenu() {
+        if (!this.chatWindow.threadViewer) {
+            return;
+        }
+        this.chatWindow.threadViewer.addComponentHint('home-menu-toggled');
```

Comment:
I would specify different hints for hide and show, it doesn't seem obvious or natural that both cases should be handled the same way.

Reply:
PS: I'm actually wondering if the bus listen shouldn't actually be done at model level too, for every threadViewer. (done at create/start/init and undone at destroy, similar to messaging/handlers bus events)

Reply:
I can't answer to your PS but I can unmerge hints